hell_charts/templates/service.yaml

```diff
@@ -1,13 +1,9 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ .Release.Name }}-service
+  name: {{ .Release.Name }}-postgresql
 spec:
-  type: NodePort
-  selector:
-    app: {{ .Chart.Name }}
   ports:
-    - protocol: TCP
-      port: 3001
-      targetPort: 3001
-      nodePort: 30100
```

Comment:
@Sidharth-Singh10 I guess he was supposed to create a new yaml file for this stuff? or am I missing something here?

Reply:
Neh, you are right on spot, he was  supposed to create a new file.